shelley/chain-and-ledger/executable-spec/src/STS/Bbody.hs

```diff
@@ -45,6 +46,7 @@ instance
 
   data PredicateFailure (BBODY hashAlgo dsignAlgo kesAlgo)
     = WrongBlockBodySizeBBODY
+    | InvalidBodyHashBBODY
```

Comment:
please also add this as a potential failure to the formal spec

Reply:
👍